<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.1">
    <title>Links for odoo14-addon-stock-exception</title>
  </head>
  <body>
    <h1>Links for odoo14-addon-stock-exception</h1>
    <a href="https://files.pythonhosted.org/packages/f0/12/5e4f30562ee872b29fef684f8e5a8c5c829f43656f1e81d7ab9dbe9dea47/odoo14_addon_stock_exception-14.0.1.0.1-py3-none-any.whl#sha256=2b1c088f1a601093312535d7f7e16697a1470202a66e5015a0d5d75ef497a81b" data-requires-python="&gt;=3.6" >odoo14_addon_stock_exception-14.0.1.0.1-py3-none-any.whl</a><br />
    <a href="https://files.pythonhosted.org/packages/27/25/4186868504498ed0cbab423e622b7b83db3acb5b0c2e0b81d3d6f3df5e8e/odoo14_addon_stock_exception-14.0.1.0.2-py3-none-any.whl#sha256=42dbf6142507f5bc6c23e3c56fd490f1f8cf1ce0033559c54b89e7289d1d725c" data-requires-python="&gt;=3.6" data-dist-info-metadata="sha256=a9ea7d7fdc809fb1aebc1ddd6bf3730f101a94c3b57dba2968cb5a914766fcce" data-core-metadata="sha256=a9ea7d7fdc809fb1aebc1ddd6bf3730f101a94c3b57dba2968cb5a914766fcce">odoo14_addon_stock_exception-14.0.1.0.2-py3-none-any.whl</a><br />
    </body>
</html>
<!--SERIAL 18659603-->